docs/nodes/bridge-node.mdx

```diff
@@ -173,29 +192,28 @@ Here is an example of starting the bridge node:
 celestia bridge start --core.ip https://limani.celestia-devops.dev/ --p2p.network arabica
 ```
 
-</TabItem>
-</Tabs>
-````
-
-You can create your key for your node by following the `cel-key` instructions [here](../developers/celestia-node-key.mdx)
+You can create your key for your node by following the `cel-key` instructions [here](../../developers/celestia-node-key)
 
 Once you start the Bridge Node, a wallet key will be generated for you.
 You will need to fund that address with Testnet tokens to pay for
-PayForData transactions.
+`PayForBlob` transactions.
 You can find the address by running the following command:
 
 ```sh
 ./cel-key list --node.type bridge --keyring-backend test --p2p.network <network>
 ```
 
-You have two networks to get testnet tokens from:
+You can get testnet tokens from:
 
-* [Arabica](./arabica-devnet.md)
-* [Mocha](./mocha-testnet.md)
```

Comment:
Why are removing Mocha from here?

Reply:
because there is now a tab for each network